modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -105,37 +108,79 @@
 
     private String urlBase;
 
+    private GenericContainer proxy;
+
     public CouchbaseContainer() {
-        super("couchbase/server:" + VERSION);
+        this("couchbase/server:" + VERSION);
     }
 
     public CouchbaseContainer(String containerName) {
         super(containerName);
+
+        withNetwork(Network.SHARED);
+        withNetworkAliases("couchbase-" + Base58.randomString(6));
+        setWaitStrategy(new HttpWaitStrategy().forPath("/ui/index.html"));
```

Comment:
we need a better wait strategy here but this is for another pr :)

Reply:
👍